webdriver-spec.html

```diff
@@ -3785,6 +3789,28 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
  are provided by the proprietary <code>window.outerWidth</code>
  and <code>window.outerHeight</code> [[!DOM]] properties.
 
+<p>To <dfn>iconify the window</dfn>,
+ given an operating system level window
+ with an associated <a>top-level browsing context</a>,
+ run implementation-specific steps
+ to iconify, minimize, or hide the window
+ from the visible screen.
+ Do not return from this operation
+ until the <a>visibility state</a>
+ of the <a>top-level browsing context</a>’s <a>active document</a>
+ has reached the <a data-lt="visibility hidden">hidden</a> state.
```

Comment:
Or until the operation times out? Or will we caveat this out in the various window commands?